<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyGraph</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenPixmap"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionSearchID"/>
    <addaction name="actionSearchType"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionQt"/>
    <addaction name="actionWriter"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomIn"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>227</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>227</width>
     <height>400</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>操作栏</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>229</width>
       <height>329</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>伸缩</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_out">
          <property name="text">
           <string>-</string>
          </property>
          <property name="shortcut">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_in">
          <property name="text">
           <string>+</string>
          </property>
          <property name="shortcut">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>旋转</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_left">
          <property name="text">
           <string>左旋</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_right">
          <property name="text">
           <string>右旋</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>设置Node颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_colorNode">
          <item>
           <property name="text">
            <string>default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>darkGray</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>yellow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>white</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>black</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>green</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cyan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>gray</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>magenta</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>设置Edge颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_colorEdge">
          <item>
           <property name="text">
            <string>default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>yellow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>white</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>black</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>green</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cyan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>gray</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>magenta</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>线宽</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_line">
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>点半径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_node">
          <property name="sliderPosition">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>显示Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_hide">
          <property name="text">
           <string>隐藏所选</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_show">
          <property name="text">
           <string>显示所有</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_shuffle">
        <property name="text">
         <string>随机振动</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>选择VTK算法</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>CircularLayout</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fast2DLayout</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ForceDirectedLayout</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PassThroughLayout</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RandomLayout</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Simple2DLayout</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Icon/6.ico</normaloff>:/image/Icon/6.ico</iconset>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Icon/Save File.ico</normaloff>:/image/Icon/Save File.ico</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Icon/Zoom In.ico</normaloff>:/image/Icon/Zoom In.ico</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/Icon/Zoom Out.ico</normaloff>:/image/Icon/Zoom Out.ico</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionOpenPixmap">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="actionSearchID">
   <property name="text">
    <string>搜索NodeID</string>
   </property>
  </action>
  <action name="actionSearchType">
   <property name="text">
    <string>搜索NodeType</string>
   </property>
  </action>
  <action name="actionWriter">
   <property name="text">
    <string>关于作者</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>关于开发</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
